modules/configuring-localnet-switched-topology.adoc

```diff
@@ -72,21 +72,23 @@ spec:
         allow-extra-patch-ports: true
         options:
           stp: false
+          mcast-snooping-enable: true <4>
         port:
-        - name: eth1 <4>
+        - name: eth1 <5>
     ovn:
       bridge-mappings:
-      - localnet: localnet2 <5>
-        bridge: ovs-br1 <6>
-        state: present <7>
+      - localnet: localnet2 <6>
+        bridge: ovs-br1 <7>
+        state: present <8>
 ----
 <1> The name for the configuration object.
 <2> A node selector that specifies the nodes to apply the node network configuration policy to.
 <3> A new OVS bridge, separate from the default bridge used by OVN-Kubernetes for all cluster traffic.
```

Comment:
```suggestion
<1> Specifies the name of the configuration object.​
<2> Specifies a node selector that identifies the nodes to which the node network configuration policy applies.​
<3> Specifies a new OVS bridge that operates separately from the default bridge used by OVN-Kubernetes for cluster traffic.​
```
Just a writing suggestion to improve [Callout in code blocks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks).

Reply:
+1